### PR TITLE
Incentive Tooltip + Infos

### PR DESCRIPTION
**Pull request by nick134-bit**
## Description and Motivation 

added tooltips, added incentives to provide liquidity.
Not sure if this fits, let me know if its too much.

incentives1

incentives3
incentives2
Incentivized5
incentivized4

## Related Issues

#295 

---

## Checklist:

- [ ] I have read [Migaloo's contribution guidelines](https://github.com/White-Whale-Defi-Platform/migaloo-frontend/blob/main/CONTRIBUTING.md).
- [ ] My pull request has a sound title and description (not something vague like `Update index.md`)
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation.
- [ ] The code is formatted properly `yarn lint`.
- [ ] The project builds and is able to deploy on Netlify `yarn build`.

### CODE REVIEW

File: components/InfoTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { InfoOutlineIcon } from '@chakra-ui/icons'
```
Comment (0xFable) [inline components/InfoTooltip.tsx:1]: More for above file 
do you think we should remove the husky stuff for now ? 
I added it to help ensure people are linting and doing some code quality like ensuring the project builds for netlify but we have gotten very good at this already. 

Any thoughts @worrex

**Review by 0xFable — DISMISSED** [review]
LGTM, one comment for discussion but the PR itself looks fine. 🚀

File: components/InfoTooltip.tsx (hunk 1 above)
Comment (nick134-bit) [reply components/InfoTooltip.tsx:1]: i think its good to have, i just have problems with pushing with github desktop and deleting fixes this. sometimes i forget to exclude it from the commit.

Comment (worrex) [reply components/InfoTooltip.tsx:1]: I think it's good to have that qa layer

Comment (worrex) [reply components/InfoTooltip.tsx:1]: @0xFable I think the flow creator can close their incentive flow anytime, can't they ?

File: components/Pages/Liquidity/DepositForm.tsx
Comment (worrex) [inline components/Pages/Liquidity/DepositForm.tsx:1]: @nick134-bit ('sentFunds: invalid coins')) {
        return 'Insufficient Funds' I think that's not correct, this error shouldn't ever occur

File: components/InfoTooltip.tsx (hunk 1 above)
Comment (worrex) [reply components/InfoTooltip.tsx:1]: @nick134-bit  can you do proper capitalization for the tooltips

**Review by 0xFable — APPROVED** [review]

**Pull request merged**